### DISCUSSION TITLE
황다영 1회차 블로그 피드백 요청드립니다!

### DISCUSSION THREAD

**Discussion by dayoungh** [Orm-camp/blog-feedback#8] [동글동글 2기 피드백]
https://dayoung-archive.tistory.com/22 

1차 미니프로젝트 회고글 입니다!
정말 기본적인 것만 써서 부족하지만,, 피드백 부탁드립니다 감사합니다 🙃

**Comment by juuyeoon** [comment]
다영님, 임주연 입니다! 회고글 작성하시느라 고생하셨어요 🙌🏻
글 마지막에 있는 다영님의 소감을 읽으며 다영님의 생각과 고민에 놀라기도 하고, 멋지다는 생각도 들었습니다!
'같이 일하기 좋은 사람이 되려면 어떻게 해야할까?' 라는 고민을 하셨다니... 그 마음가짐을 가지고 계신 것부터가 좋은 팀 구성원의 자세가 아닐까 싶네요! 😊
글 내부에서 구현기능을 파트별로 나누어서 기술해 주셔서 편하게 글이 잘 읽혔다고 생각했습니다!
너무 고생하셨어요 🤗

**Comment by Yu-Kyeong-Seok** [comment]
블로그 글 쓰신게 있어서 그런지 재밌게 봤습니다!
마지막 멘트들이 뭔가 일기 느낌 나서 좋았습니다 😊

**Comment by taeyoung0001** [comment]
깔끔하게 잘 쓰셨네요 ㅎㅎ
'같이 일하기 좋은 사람이 되려면 어떻게 해야할까?'라는 질문에 저도 답해보고 싶습니다!

제 생각으론
1. 긍정적인 에너지를 가지고 일하는, 밝은 사람
2. 누군가의 피드백을 열린 마음으로 받아드리고 자기 발전을 위해 건설적인 사람
3. 맡은 바에 책임감을 가지고 원할한 의사소통과 협업 능력을 가진 사람이라고 생각합니다! 

왠지 다영님은 위에 3가지 다 가지셨을것 같네요!!

**Comment by hyyun1113** [comment]
다영님! 블로그 글 잘읽었습니다. 이번 미니플젝 같은 팀원이었던 저한테 다영님은 충분이 같이 일하기 좋은 팀원이었습니다!! 
블로그 글은 전반적으로 무척 읽기 편했습니다. 다만 모바일에러인지 아니면 의도하신건진 모르겠지만 기술스택부터 회고까지, 리스트 항목 시작의 동그라미 기호라고 할까요? 그 부분이 두개씩 나오고 있습니다.
이 외에는 블로그 글 너무 좋았습니다!

**Comment by ggaong** [comment]
다영님 글 잘 읽었습니다! 🙌🙌

이미지와 코드가 들어가니 좀 더 눈에 쏙쏙 들어오는 것 같습니다!
마지막 소감에서도 '같이 일하기 좋은 사람이 되려면 어떻게 해야할까' 라는 고민을 하셨다는게 이미 좋은 팀원이 아닐까..!! 라는 생각이 듭니다😄😄

글 쓰시느라 너무 고생많으셨습니다 😆

**Comment by euneunsol** [comment]
다영님! 회고글 너무 잘 읽었습니다!
이슈사항과 해결 부분에 이미지로 어떤 이슈가 있었는지 한눈에 알아볼 수 있어서 좋은것 같아요!
마지막 부분에서 다른사람이 보기에도 명확한 코드를 짜는게 힘드셨다고 했는데 저 역시도 같은 경험을 해서 너무 공감이 되네요🥲 글 쓰시느라 수고 많으셨어요!😊

**Comment by ehddk** [comment]
다영님의 자세한 회고록 너무 잘 읽었습니다 :ㅇ
기본적인 것만 쓰신게 아닌 것 같은데요!
이슈- 해결방법 -시도 까지 기재해주셔서 프로젝트를 정말 열심히 임해주셨다는게 느껴지네요 . 
'공유할 줄 아는 사람이 같이 일하기 좋은 사람'이라는 말씀에 매우 공감해요 !
